snapshot.sh

```diff
@@ -31,9 +32,15 @@ while true; do
         aws_args="--endpoint-url ${S3_HOST}"
         s3_uri_base="s3://${SNAPSHOT_PATH}"
         timestamp=$(date +"%Y-%m-%dT%H:%M:%S")
-        s3_uri="${s3_uri_base}/${SNAPSHOT_PREFIX}_${timestamp}.tar.gz"
+        s3_uri="${s3_uri_base}/${SNAPSHOT_PREFIX}_${timestamp}.${SNAPSHOT_FORMAT}"
 
-        tar c -C $SNAPSHOT_DIR . | gzip | aws $aws_args s3 cp - "$s3_uri" --expected-size $SNAPSHOT_SIZE
+        SNAPSHOT_SIZE=$(du -sb $SNAPSHOT_DIR | cut -f1)
```

Comment:
Should we even allow user to set `SNAPSHOT_SIZE` if we override it anyway...

Reply:
Good point, gonna update the PR

Reply:
As well as mention SNAPSHOT_FORMAT in the readme file.